topics/epigenetics/tutorials/cut_and_run/tutorial.md

```diff
@@ -264,7 +245,7 @@ repetitive regions but keep reads falling into regions present in alternate loci
 >
 > > <solution-title></solution-title>
 > >
-> > 41.46+57.51=98.97%
+> > 36.47+62.39=98.86%
```

Comment:
I don't know why these number changed (maybe the version of bowtie2)...

Reply:
Yeah this can happen with different versions of alignment tools